spec/unit/postmark/account_api_client_spec.rb

```diff
@@ -332,20 +334,20 @@
       }
 
       it 'performs a GET request to /domains endpoint' do
-        allow(subject.http_client).to receive(:get).
+        expect(subject.http_client).to receive(:get).
             with('domains', :offset => 0, :count => 30).
             and_return(response)
         subject.get_domains
```

Comment:
my only on this one would be that although it is better, the part which I don't like is that we don't have assets at the end of the tests. They should always be at the end, otherwise that last call (in this case subject.get_domains) , kind of has no meaning

Reply:
> They should always be at the end

I find it easier to follow when they're at the end as well! 

That would take a bit more of a reworking of the mocking approach currently employed by this test particular suite, but I prefer that as well when possible. 🙂